tests/drivers/eeprom/api/boards/s32z2xxdc2_s32z270_rtu0.overlay

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		eeprom-0 = &eeprom0;
+	};
+};
+
+&lpi2c2 {
+	pinctrl-0 = <&i2c2_default>;
+	pinctrl-names = "default";
+	status = "okay";
+
+	eeprom0: eeprom@50 {
```

Comment:
If this EEPROM is present on the board, it needs to be added to the board itself. If not, it should not be added here solely for your testing as this will then fail for everyone else.

Reply:
i updated, move it into boards

Reply:
the EEPROM is present on the board but by default it's is used to hold configuration used by the boot ROM. Therefore, I enable it in the test for internal testing purpose only